python-avd/pyavd/_eos_designs/schema/schema_fragments/underlay_isis_authenticatin_cleartext_key.schema.yml

```
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  underlay_isis_authentication_cleartext_key:
+    documentation_options:
+      table: isis-settings
+    type: str
+    description: |-
+      Cleartext password. `underlay_isis_authentication_key` takes precedence over this key.
```

Claus asked to not have the same precedence mentions in both keys on a recent PR of mine:

```suggestion
      Cleartext password.
```

Move the precedence to the existing key `underlay_isis_authentication_key` and write it like it was done for others please for consistency.

---

Done.